File: banda/IvanRC-banda-Java/src/main/java/modelsBandaMusicos/Musico.java
package modelsBandaMusicos;

public class Musico extends Persona {

    String nombre;
    Double salario = 1500.00;
    int añosExperiencia;

    public Musico(String nombre, int añosExperiencia){
        this.nombre = nombre;
        this.añosExperiencia = añosExperiencia;
    }

    public void interpretar(){
        System.out.println("Músico: Soy "+nombre+" y estoy interpretando...");
    }
}

File: banda/MarioBueno_Banda_Kotlin/src/main/kotlin/models/Musico.kt
//Mario Bueno López
//maarioo2525@gmail.com

package models

open class Musico(override var nombre: String?, open var experiencia: Int) : Persona() {

    open fun interpretar() {
        println("Estoy interpretando")
    }
    companion object{
        var salario = 1500.00
    }
}